lib/api/model/model.dart

```diff
@@ -664,6 +665,37 @@ enum MessageFlag {
   String toJson() => _$MessageFlagEnumMap[this]!;
 }
 
+@JsonSerializable(fieldRename: FieldRename.snake)
+class MessageEditHistory {
```

Comment:
nit: Let's put this at the end of this file, after `DmMessage`. That way it doesn't get between reading `Message` and reading `StreamMessage`, or `DmMessage`.

(Most of the time, the edit history isn't part of what one needs to think about when thinking about a message.)